src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((result, style) => {
+    const [prop, value] = style.split(':').map((x) => x.trim());
+
+    if (prop && value) {
+      result[prop] = value;
```

Comment:
The check for both 'prop' and 'value' is good to avoid adding invalid entries to the result object. Make sure to handle cases where 'value' might contain important CSS flags like '!important', as the current logic will include these flags as part of the value without any special handling.